templates/availability/service-availability-editor.phtml

```diff
@@ -131,24 +128,36 @@
                         </div>
                     </template>
 
-                    <div class="form-row">
-                        <div class="form-row__label">Ends</div>
+                    <div :class="['form-row', errors.has('ends_after') ? 'form-row--with-error' : '']">
+                        <div class="form-row__label">
+                            Ends
+                            <div class="form-row__description form-row__error" v-if="errors.has('ends_after')">{{ errors.first('ends_after').replace('_', ' ') }}</div>
+                        </div>
                         <div class="form-row__input">
                             <div>
                                 <label for="ends-after" style="width: 90px; display: inline-block">
-                                    <input type="radio" id="ends-after" value="period" v-model="model.repeatUntil"> After
+                                    <input type="radio" @change="errors.remove('ends_after')" id="ends-after" value="period" v-model="model.repeatUntil"> After

```

Comment:
Later, the validation rules should come from a centralized location on the server. See [card](https://github.com/orgs/RebelCode/projects/2#card-11147620).